### DISCUSSION TITLE
Rule for enforcing property ordering

### DISCUSSION THREAD

**Discussion by land-cap** [chakra-ui/eslint-plugin-panda#167] [Ideas]
Tailwind's editor extensions have a great feature that automatically orders classes so there is always consistency and you know where to look for certain styles in any component. Would be great if we had that in Panda. Sometimes when there's more properties it's getting hard to keep a consistent rule for ordering and takes some cognitive load.

**Comment by anubra266** [comment]
@land-cap Have you tried this prettier plugin?
https://www.npmjs.com/package/@pandabox/prettier-plugin

Sorting is a prettier task, not linting

**Reply by land-cap** [reply]
Indeed, that's what I need. You guys are doing a great job pushing this project further🚀 This was probably the main frustration for me.
